rcl/src/rcl/guard_condition.c

```diff
@@ -133,7 +133,7 @@ rcl_guard_condition_get_default_options()
 }
 
 rcl_ret_t
-rcl_trigger_guard_condition(const rcl_guard_condition_t * guard_condition)
```

Comment:
Actually, they don't undo master, but they do the same as master. I think they'll just disappear on merge, but if you have an opportunity to rebase before the final CI run, you can get rid of this commit that way.